node/bin/src/prover_api/gapless_l1_prove_sender.rs

```diff
@@ -0,0 +1,66 @@
+use async_trait::async_trait;
```

Comment:
Metacomment: Does this component really belong to prover_api? I'd assume it might be more suitable for L1 interactions (maybe sender?)